code/_globalvars/lists/color.dm

```diff
@@ -0,0 +1 @@
+GLOBAL_LIST_INIT(contrast_colors, list("#fF0000", "#b900f7", "#00a2ff", "#ff7b00", "#000000"))
```

Comment:
Could you use defines and add them to `colors.dm`?

Reply:
you're addicted to defines